gnocchi/common/redis.py

```diff
@@ -73,7 +73,7 @@
 
 def get_client(conf):
     if redis is None:
-        raise RuntimeError("python-redis unavailable")
+        raise RuntimeError("Redis Python module is unavailable")
```

Comment:
why not just specify the exact lib?

Reply:
It's `redis`. If you can find a way to say that without being confusing about the redis-server, please halp meeeee

Reply:
*shrugs*